src/index.js

```diff
@@ -0,0 +1,4 @@
+import React from 'react'
+import { render } from 'react-dom'
+
+render(React.createElement('div', {id: 'MyFirst'}, 'My first component'), document.getElementById('root'))
```

Comment:
I would recommend to use JSX syntax instead of `React.createElement` API. Also, part of the configuration is to provide a separate components that owns receives data and owns css. My recommendation is to create an App component that injects another component and provides data to it.